tests/runtimes/test_run.py

```diff
@@ -76,6 +76,21 @@ def test_new_function_from_runtime():
     )
 
 
+def test_new_function_default_foo():
+    runtime = _get_runtime()
+    function = mlrun.new_function(runtime=runtime)
+    expected_runtime = runtime
+    expected_runtime["spec"]["foo"] = mlrun.mlconf.function.spec.foo.default
```

Comment:
check other default values as well. since here the default is `None` it will be useful to set it to some other default

Reply:
Good thinking! I added one test with a default override. Let me know if that's what you had in mind.